rsmtool/notebooks/true_score_evaluation.ipynb

```diff
@@ -55,8 +55,8 @@
     "                        \"and double-scored responses, and mean squared error (MSE) and \"\n",
     "                        \"proportional reduction in mean squared error (PRMSE) for \"\n",
     "                        \"predicting a true score with system score. As for other evaluations, \"\n",
-    "                        \"these results are computed on the evaluation set. Trim (bound) scores \"\n",
-    "                        \"are truncated to [{}, {}]. Trim-round scores are computed \"\n",
+    "                        \"these results are computed on the evaluation set. `raw_trim` scores \"\n",
+    "                        \"are truncated to [{}, {}]. `raw_trim_round` scores are computed \"\n",
     "                        \"by first truncating and then rounding the predicted score. Scaled scores \"\n",
```

Comment:
Same as above: we have both raw_ and scale_

Reply:
This talks about the scaled scores separately in the next sentence so we don't need to change this.